reservation/test_typeOfJob_addList.py

```diff
@@ -15,12 +15,14 @@ def save_type_of_job(make_typeOfJob):
 @pytest.mark.django_db
 class TestTypeOfJob:
     def test_typeOfJob_list(self, client, professional):
-        response = client.get(reverse('typeOfJob', kwargs={'professional': professional.pk}))
+        client.force_login(professional.profile_id.user_id)
+        response = client.get(reverse('typeOfJob'))
         assert response.status_code == 200
         assert 'reservation/typeOfJob_list.html' in response.templates[0].name
```

Comment:
Do'nt you think you could add a test for checking there is content?
https://github.com/redhat-beyond/Pro4U/pull/129/files#diff-e5871311eb0c98a9a7e45d4efd8addcc267d4ec3ada9c3eed75d0cf73940b8bfR24

Reply:
Done @Yarboa